std-bits/base/src/main/java/org/enso/base/enso_cloud/logging/RequestFailureException.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.base.enso_cloud.logging;
+
+public final class RequestFailureException extends RuntimeException {
```

Comment:
- Make package private
- or use somewhere outside of this package
- not sure why to have a special exception at all
- `IOException` and subclasses would be good enough, in my opinion.

Reply:
Done in 039b27949d88dee15ef785f348fff2c4fe329043